test/Microsoft.AspNet.Mvc.Razor.Test/RazorViewTest.cs

```diff
@@ -555,11 +555,13 @@ public class RazorViewTest
                     await v.RenderSectionAsync("sectionA");
                 });
             });
+            nestedLayout.Path = "NestedLayout";
```

Comment:
Yeah. In the general case, we set the `IRazorPage.Path` to the location where we found the cshtml file so it shouldn't be null.